test/GitcoinGovernor.t.sol

```diff
@@ -117,10 +117,19 @@ abstract contract ProposalTestHelper is GitcoinGovernorTestHelper {
   function setUp() public virtual override {
     GitcoinGovernorTestHelper.setUp();
 
-    initialProposalCount = governorAlpha.proposalCount();
-
-    ProposeScript _proposeScript = new ProposeScript();
-    upgradeProposalId = _proposeScript.run(governorBravo);
+    if (_useDeployedGovernorBravo()) {
+      // The actual upgrade proposal submitted to Governor Alpha by kbw.eth on August 9th, 2023
+      upgradeProposalId = 65;
+      // Since the proposal was already submitted, the count before its submissions is one less
+      initialProposalCount = governorAlpha.proposalCount() - 1;
+    } else {
+      initialProposalCount = governorAlpha.proposalCount();
+      ProposeScript _proposeScript = new ProposeScript();
+      // We override the deployer to use kevinolsen.eth, because in this context, kbw.eth already
+      // has a live proposal
+      _proposeScript.overrideProposerForTests(0x4Be88f63f919324210ea3A2cCAD4ff0734425F91);
```

Comment:
Alternative idea to avoid needing to expose a custom method and break the scopelint checks is just use the `StdStorage` library here from forge-std

Reply:
Found an alternative way to make the linter happy (extending the propose script for our tests). See 9f0378b4fd4cf7743a2259b7364d035b15ca2734